iree/hal/vulkan/vulkan_driver_module.cc

```diff
@@ -78,7 +78,7 @@ StatusOr<ref_ptr<Driver>> CreateVulkanDriver() {
 
   if (absl::GetFlag(FLAGS_vulkan_validation_layers)) {
     options.instance_extensibility.optional_layers.push_back(
-        "VK_LAYER_LUNARG_standard_validation");
+        "VK_LAYER_KHRONOS_validation");
```

Comment:
Can you update the other instances of this name too?

https://github.com/google/iree/blob/14257a619ffec02d02a618aeceac4786c09d36b5/iree/hal/vulkan/api.h#L51

https://github.com/google/iree/blob/99913e489ed7271f95253ab53188db8976ef1bd0/iree/hal/vulkan/api.cc#L91-L93